src/core/evaluator.js

```diff
@@ -2281,6 +2281,7 @@ class PartialEvaluator {
     viewBox,
     markedContentData = null,
     disableNormalization = false,
+    keepWhiteSpace = false,
```

Comment:
If you change the signature of this method then you must ensure that the new parameter is handled *consistently* at all the relevant call-sites in this file (see below).
As-is, this could otherwise cause a (preventable and unnecessary) future bug in this code.

https://github.com/mozilla/pdf.js/blob/dc01782633432642fe8b56e3c25bdc9a430d9e02/src/core/evaluator.js#L3263-L3275